packages/epo-react-lib/.env.development

```diff
@@ -0,0 +1,2 @@
+STORYBOOK_ENV=true
+STORYBOOK_PUBLIC_BASE_URL=http://localhost:6006
```

Comment:
need to add env variables to stand in for the next.js BASE_URL vars expected by some components